Doc/library/typing.rst

```diff
@@ -1230,6 +1232,123 @@ These are not used in annotations. They are building blocks for creating generic
     ``covariant=True`` or ``contravariant=True``.  See :pep:`484` for more
     details.  By default, type variables are invariant.
 
+.. class:: TypeVarTuple
+
+    Type variable tuple. A specialised form of :class:`Type variable <TypeVar>`
+    that enables *variadic* generics.
+
+    A normal type variable enables parameterization with a single type. A type
+    variable tuple, in contrast, allows parameterization with an
+    *arbitrary* number of types by acting like an *arbitrary* number of type
+    variables wrapped in a tuple. For example::
+
+        T = TypeVar('T')
+        Ts = TypeVarTuple('Ts')
+
+        def remove_first_element(xs: tuple[T, *Ts]) -> tuple[*Ts]:
```

Comment:
Any reason why you went for `xs` as the variable name? Maybe `tup` or `var` would be slightly friendlier for readers? :)

(Sorry, I'm being really picky now!!)

Reply:
Nah, no strong reason. Changed to `tup` :)